### DISCUSSION TITLE
Guest users unable to use Webpart

### DISCUSSION THREAD

**Discussion by Chris-Magenium** [microsoft-search/pnp-modern-search#2897] [Q&A]
We have installed v4.8 and our guest users are getting the following error:

<img width="207" alt="image" src="https://user-images.githubusercontent.com/129423239/230431501-b10632b0-34a6-435b-8ff0-e7f63e338fa2.png">

I've read through the discussion from December of last year where a link was provided to a blog about allowing guest users read access to the App Catalog.  We are not comfortable with that solution but would be fine allowing it possibly at the app level.  I'm having a hard time finding out how to do that at the app level.  Any help is appreciated.

**Comment by kasperbolarsen** [comment]
Are the guest users able to see the content when using out of the box MS Search?

**Reply by Chris-Magenium** [reply]
If they are in the specific site where the document lives they can find it through search with out of the box MS Search.  We created a site with the webparts to have a more customizable search experience from one location.  This is where the problem comes in.

**Comment by wobba** [comment]
You will likely find your solution in @LauraKokkarinen awesome blog post https://laurakokkarinen.com/sharepoint-online-guest-user-troubles-and-how-to-get-past-them/
